hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/missingContainers/missingContainers.tsx

```diff
@@ -341,6 +341,22 @@ export class MissingContainers extends React.Component<Record<string, object>, I
     return <div>Loading...</div>;
   };
 
+  serachCoulmn = () => {
```

Comment:
searchColumn?

Reply:
Couldn't we move the whole `Table` into the new method? Something like this:
```
generateTable = (dataSource: IContainerResponse[]) => {
  <Table
    expandRowByClick dataSource={dataSource}
    columns={CONTAINER_TAB_COLUMNS.reduce<any[]>((filtered, column) => {
      if (column.isSearchable) {
        const newColumn = {
          ...column,
          ...new ColumnSearch(column).getColumnSearchProps(column.dataIndex)
        };
        filtered.push(newColumn);
      } else {
        filtered.push(column);
      }
  
      return filtered;
    }, [])}
    loading={loading}
    pagination={paginationConfig} rowKey='containerID'
    expandedRowRender={this.expandedRowRender} onExpand={this.onRowExpandClick}/>
}
```